RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Live.swift

```diff
@@ -59,34 +59,38 @@ extension FactorSourcesClient: DependencyKey {
 			return factorSourceID
 		}
 
-		return Self(
-			getCurrentNetworkID: {
-				await profileStore.profile.networkID
-			},
-			getMainDeviceFactorSource: {
-				let sources = try await getFactorSources()
-					.filter { $0.factorSourceKind == .device && !$0.supportsOlympia }
-					.map { try $0.extract(as: DeviceFactorSource.self) }
+		let getMainDeviceFactorSource: GetMainDeviceFactorSource = {
```

Comment:
hard to read without "split" mode in Github... I've just moved `getMainDeviceFactorSource` into a variable....